src/App.vue

```diff
@@ -448,6 +448,15 @@ export default class App extends Mixins(
       console.log('Error fetching user info or updating Launch Darkly =', error)
     }
 
+    // now that LaunchDarkly has been updated (ie, in case of user targeting),
+    // check whether to use this Entity Dashboard or the new Business Dashboard
+    if (GetFeatureFlag('use-business-dashboard')) {
+      const identifier = (this.businessId || this.tempRegNumber)
+      const dashboardUrl = `${this.getBusinessDashUrl}/${identifier}${this.$route.fullPath}`
+      navigate(dashboardUrl)
+      return
+    }
+
```

Comment:
I hoped to add this code to main.ts, ie, before Vue was started, but for FF targeting, the LD user first had to be updated (line 443).

This code redirects to the new business dashboard if the FF is True. This means we don't have to update Namerequest UI or Auth Web (My Business Registry) to redirect conditionally.

Note that `$route.fullPath` includes any URL parameters (such as Filing ID and Account).

If you want to test this, contact me with your IDIR and I'll update LaunchDarkly to target you only.

Reply:
Sample redirect URL when :

![image](https://github.com/user-attachments/assets/c43f3324-9a24-46a8-8fab-4a773c9b8622)